aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/data/AmplifyCredential.kt

```diff
@@ -95,16 +96,22 @@ internal data class FederatedToken(val token: String, val providerName: String)
  */
 @Serializable
 internal data class CognitoUserPoolTokens(
```

Comment:
What does the toString look like now? I think the tokens are still tracked properly after traversing through the code, but notice expiration is now included (probably for the best).

Reply:
I had not noticed that expiration was excluded before, but that seems like an error. The updated toString looks like this:

```
CognitoUserPoolTokens(idToken=eyJh***, accessToken=eyJh***, refreshToken=eyJh***, expiration=1758044817)
```